[CS2113T-T12-4] Scheduler--;

Colin386 opened the pull request:
Scheduler--; (S--) is a desktop app for managing deadlines from different sources. The user will use a Command Line Interface to control it. It is specially designed for Computing students who are comfortable in using CLI and have Git project deadlines as well as consolidated Zoom session links due which will suit home based learning in this COVID period.

iamchenjiajun commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+#### Command Component
+
+![Diagram for commmand](./diagrams/Command.jpg)
```
Was it necessary to show all the `Command` classes?

According to the CS2113 website,
> Omit repetitive details e.g., a class diagram can show only a few representative ones in place of many similar classes (note how the AB3 Logic class diagram shows concrete `*Command` classes).

OngDeZhi replied:
![image](https://user-images.githubusercontent.com/39303087/97659754-bfd37180-1aaa-11eb-9521-717b5e4b41fc.png)

You may want to consider using XYZCommand to represent all of these classes, it is a little lengthy.

iamchenjiajun commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+API: [Command.java](https://github.com/AY2021S1-CS2113T-T12-4/tp/blob/master/src/main/java/seedu/duke/command/Command.java) 
+1. `Parser` class to parse the user command 
+1.  This results in a `Command` object executed by the `Parser`
```
Is the `Command` being executed by the `Parser` or `Duke`? According to one of the sequence diagrams below, it seems that `Duke` calls the `execute()` method on `Command`.

![image](https://user-images.githubusercontent.com/1352844/97533177-b686ce00-19f2-11eb-8c62-d5b117d4791a.png)

iamchenjiajun commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+The repeat command first locates the event that is to be repeated along with its starting date.
+Next, it creates an empty ArrayList called `repeatEventList` that will contain all repeated events.
+
+![Sequence Diagram for Repeat Command step 1](./diagrams/repeatstep1.jpg)
```
Are the arrow heads and tails supposed to connect here?

![image](https://user-images.githubusercontent.com/1352844/97534151-77597c80-19f4-11eb-8e82-99ccf7c050e3.png)

jzihao97 replied:
![image](https://user-images.githubusercontent.com/57383789/97660563-eeeae280-1aac-11eb-8fee-458b5dc1ba91.png)

Consider connecting the arrows to the activation bar.

iamchenjiajun commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+The following sequence diagram shows how the check operation works:
+
+![Sequence Diagram for CheckCommand](./diagrams/CheckCommand_seq_diagram.jpg)
```
Maybe a more descriptive return value can be written here instead of `c`.

![image](https://user-images.githubusercontent.com/1352844/97534366-d15a4200-19f4-11eb-8125-19c42d1f6954.png)

iamchenjiajun commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+This sequence diagram shows how the `getDate` method functions:
+
+![Sequence Diagram for getDate](./diagrams/getDate_seq_diagram.jpg)
```
Could this diagram be split into smaller parts? Especially with so many sections of the `alt` frames, it makes the diagram a little difficult to comprehend.

iamchenjiajun reviewed (commented):
Overall well done! Some parts could be improved but I like the Architecture section where the diagrams were clearly used to explain the interactions between the components.

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+It is also responsible for printing messages from commands and exception messages. 
+
+#### Parser Component
+![Parser diagram](./diagrams/Parser_diagram.jpg)
```
Perhaps a sequence diagram would be helpful here. The current diagram does not show where the Command goes to, and as such does not match the explanation given below.

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+The `Parser` object uses its `parse` method to parse inputs passed to it from `Ui`. This method returns a `Command` object to be executed by `Duke`.
+
+#### UserData Component
+![UserData diagram](./diagrams/UserData_diagram.jpg)
```
Indicating multiplicities may be helpful. Also, it may be better to omit the aggregation relationship, according to the textbook, since it does not really add anything of value in my opinion.
![image](https://user-images.githubusercontent.com/62279011/97543809-b17e4a80-1a03-11eb-9a8a-e2a67330b45e.png)

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+- does not depend on the other components.
+
+#### Storage Component
+![Diagram for storage class](./diagrams/classDiagramStorage.jpg)
```
The diagram implies that storage knows about UI but UI does not know about storage. Should change to double-headed arrow if both are aware of each other, or reverse the direction of the arrow if UI knows about Storage.
Also, consider replacing the 4 classnames connected to Storage with the actual classnames. For the class attribute name, consider using the standard notation of name:type = default-value e.g. name:String = personal.txt. Unless the 4 classes are not actually classes, in which case they should not be shown on a class diagram as it is non-standard notation.

Consider having some multiplicities in general.

Storage to StorageParser might be better represented by a solid line, since it seems that Storage cannot function fully without the StorageParser i.e. the dependency is not temporary.

jerroldlam replied:
![image](https://user-images.githubusercontent.com/60382285/97659056-e98b9900-1aa8-11eb-96e3-7e641d10f81c.png)
Should the classnames be with colon and underline in the titles to follow the UML notation? Also, do consider using all sharp bends or rounded bends for the arrows for consistency.

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+- is able to store all event and goal information onto the computer
+- is able to load all event and goal information from the computer into the program
+
+![Diagram for storageOverall](./diagrams/storageOverall.jpg)
```
Consider using sequence diagrams instead of non-standard diagram. More chance to showcase diagramming skills is always good. See below: some other type used ... UML ... worked just as well.
![image](https://user-images.githubusercontent.com/62279011/97544627-d7f0b580-1a04-11eb-8820-25dac4a93fbf.png)

jerroldlam replied:
![image](https://user-images.githubusercontent.com/60382285/97659226-69196800-1aa9-11eb-964c-cb18d25f0743.png)
In addition to the above comment, you can consider finding a common place to label the arrows, ie label all below / on top of the arrow as the current diagram has labels on top of and on the arrows itself. This would make the diagram look neater for the reader.

Also it is not immediately clear which you are referring to, loading or saving. Perhaps putting the loading or saving on top in the middle would help with this as putting it in the top left makes it easier to miss

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+Next, when `addCommand#execute` is called from the main, this method will call the respective method to create one of the three events. These methods are `addCommand#addPersonal`, `addCommand#addZoom`, and `addCommand#addTimetable`.
+
+The personal event can contain the following fields: 
+1. Description
+2. Description and date
+3. Description, date and time
+
+The zoom event can contain the following fields:
+1. Description and link
+2. Description, link, date and time
+
+The Timetable event can contain the following fields:
+1. Description, date and time
+2. Description, location, date and time
```
Consider using class/object diagram to represent the attributes of each event

Artemis-Hunt commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+ 
+The following sequence diagram shows how the whole add feature works: <br>
+
+![Sequence Diagram for Add Command](./diagrams/addCommand.jpg)
```
Consider adding details about what is being returned. The current diagram implies that addCommand, for example, is returning void, which is probably not the case.

Class names should have a capital first letter i.e. AddCommand and not addCommand.

Consider using a general addEvent and general :Event object to represent the 3 classes, as the method calls and invocations are identical except for the names of the object/method. (e.g. diagram drawn in week 10 lecture, where Cat and Dog were represented by a single Animal class)

jerroldlam replied:
![image](https://user-images.githubusercontent.com/60382285/97660231-e514af80-1aab-11eb-9c73-95dcba855aac.png)

Should it be AddCommand instead?

Artemis-Hunt reviewed (commented):
Good architecture diagrams. Could consider using more sequence diagrams in the earlier half. Later half could use more class diagrams. Good mix of both ensures maximum clarity

jerroldlam commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+The following sequence diagram shows the overall process of repeating an event:
+
+![Sequence Diagram for Repeat Command](./diagrams/RepeatScenario.jpg)
```
![image](https://user-images.githubusercontent.com/60382285/97659895-193ba080-1aab-11eb-9b02-eb97ff5e5c0c.png)

For the left most circle, should the return be included? The other return lines do not have them.

For the 2 right-most circles, the lifelines come out of nowhere and have no constructor for this diagram. Should  they have more information about how they are initialized as there is no indication of what classes they are. This would help greatly when a developer looks at the overall picture instead of needing to look at the references.

Should the alt box be opt instead?

jzihao97 commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+The following sequence diagram shows how the `list zoom` command is parsed:
+
+![Sequence diagram for parsing list command](./diagrams/ListParseSequenceDiagram.jpg)
```
![image](https://user-images.githubusercontent.com/57383789/97660206-d4fcd000-1aab-11eb-935f-8c609c9e3258.png)

Perhaps instead of using ref block inside the picture you can use other representation to reduce confusion with the developer.

jerroldlam commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+Given below is how the deadline command behave: <br>
+
+<p align="center">
+  <img width="414" height="562" src="./diagrams/DeadlineScenario.jpg">
```
![image](https://user-images.githubusercontent.com/60382285/97660415-7126d700-1aac-11eb-8b23-9df99d628622.png)
should the states in the state diagrams be full boxes instead of the half boxes here?

jerroldlam commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+<ul>
+    <li>has a need to organise events using scheduler</li>
+    <li>prefer desktop application</li>
```
should this be "prefers"?

jerroldlam commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+|v1.0|university student having projects|be informed of the goals of my project|keep track of the project details|
+|v1.0|university student|customise repeated task|set events like tutorials to weekly|
+|v1.0|teacher|organise all my zoom links for my classes|it can be easily accessible.|
+|v1.0|user|see the events that I have created|refer to them when I forget|
```
I think it would be best to specify user of your app as user is a really broad term and can refer to a user of anything

OngDeZhi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+Steps 2 to 5 are repeated up till the number specified by `count` as shown in the loop.
+
+![Sequence Diagram for Repeat Command step 6](./diagrams/repeatstep6.jpg)
```
![image](https://user-images.githubusercontent.com/39303087/97660673-3d987c80-1aad-11eb-9139-e862d4202c2f.png)

The arrows (at the end) are supposed to be connected I think? May want to consider joining it :)

OngDeZhi commented on docs/DeveloperGuide.md:
```diff
@@ -1,34 +1,610 @@
+
+The following sequence diagram shows how the deadline operation works: <br>
+
+![Sequence Diagram for Deadline Command](./diagrams/DeadlineSequenceDiagram.jpg)
```
![image](https://user-images.githubusercontent.com/39303087/97660801-936d2480-1aad-11eb-8dee-dc40c31cce98.png)

You may want to consider ending the dotted line before the cross.

jerroldlam reviewed (commented):
Overall still readable but some diagrams can be improved to enhance readability of the developer guide! Overall a very decent effort

OngDeZhi reviewed (commented):
Overall good job! I think maybe just a minor issue with the arrows not connecting, otherwise you guys did a great job making the diagram easy to read!

ninggggx99 committed:
Merge branch 'master' into branch-AddLogging

Reinbowl committed:
Update user guide

Colin386 committed:
Updated Help file

Colin386 committed:
Merge pull request #258 from ninggggx99/branch-AddLogging

Add logging for Deadline, Note, Reminder, View

Colin386 committed:
Merge branch 'master' into branch-helpfileupdate

Colin386 committed:
Merge pull request #260 from Colin386/branch-helpfileupdate

Branch helpfileupdate

ninggggx99 committed:
Updated the user guide with latest commands

ninggggx99 committed:
Merge pull request #261 from ninggggx99/branch-AddingViewtoUG

Updated the user guide

Colin386 committed:
Handling blank locations
Blank string saved to indicate no location.

ninggggx99 committed:
View command naming in table updated

ninggggx99 committed:
update naming

ninggggx99 committed:
Merge pull request #263 from ninggggx99/branch-AddingViewtoUG

Updating View command in UG

Colin386 committed:
Modify test case
Used for checking empty location details

Colin386 committed:
Merge branch 'master' into branch-saveTimetableFix

Reinbowl committed:
Merge branch 'master' into userguide

Reinbowl committed:
Merge pull request #264 from Reinbowl/userguide

Update user guide

Reinbowl committed:
Merge pull request #266 from Reinbowl/developerguide

Update developer guide with diagrams for list and calendar

marcursor committed:
Edit done, undone, delete commands to notify the user if input date does not match any event and detect missing event type or event index
Add Junit for done, undone, delete

marcursor committed:
Update delete command to set repeat task to non-repeat if it deletes the last repeat sub-event
Update Junits

marcursor committed:
Merge branch 'master' into branch-Junit

marcursor committed:
Fix checkstyle length of lines

ninggggx99 committed:
Merge pull request #265 from Colin386/branch-saveTimetableFix

Branch save timetable fix

Colin386 committed:
Fix logging
Logging now will only print to text file rather than the terminal

Colin386 committed:
log added to repeat command

ninggggx99 committed:
Merge pull request #269 from Colin386/branch-loggingRepeat

Branch logging repeat

matthewgani committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp into branch-UpdateDevGuide

# Conflicts:
#	docs/UserGuide.md

matthewgani committed:
Updated UG to be more consistent with other parts

matthewgani committed:
Added javadoc for extract methods and ui methods

matthewgani committed:
Merge pull request #267 from marcursor/branch-Junit

Branch junit

marcursor committed:
Edit check command to validate time period
Update junit

marcursor committed:
Fix Junit

marcursor committed:
Fix Junit

matthewgani committed:
FIx userguide to be more specific for addcommand, started adding logging and assertions

Colin386 committed:
Java doc for bye help repeat

Reinbowl committed:
Update javadoc

marcursor committed:
Merge pull request #273 from Reinbowl/javadoc

Update javadoc

Reinbowl committed:
Add logging for calendar, goal, list commands

marcursor committed:
Merge pull request #272 from marcursor/branch-CheckCommandv2.1

Edit check command to validate time period

matthewgani committed:
Added assertion and logging to add and extractcommands. Refactored a bit of extractCommand

matthewgani committed:
Merge branch 'master' into branch-UpdateDevGuide

# Conflicts:
#	src/main/java/seedu/duke/command/CheckCommand.java

Colin386 committed:
Merge branch 'master' into branch-ColinJavaDoc

matthewgani committed:
Finished assertion and logging, passed checkstyle

Reinbowl committed:
Update list junit testing

Reinbowl committed:
Merge pull request #274 from Reinbowl/logging

Add logging for calendar, goal, list commands

Colin386 committed:
Javadoc for some UI and EventLogger
UI functions related to repeat event or saving has javadoc added

Colin386 committed:
Merge branch 'master' into branch-ColinJavaDoc

Colin386 committed:
add ending period

ninggggx99 committed:
Updated DG
- View Command implementation
- Note Command implmentation
- Reminder Command implementation
- new diagram for deadline
- updated table of content

Reinbowl committed:
Merge pull request #276 from Reinbowl/junit

Update list junit testing

Colin386 committed:
Merge branch 'master' into branch-ColinJavaDoc

ninggggx99 committed:
Merge branch 'master' into branch-DGupdate

Colin386 committed:
Merge pull request #277 from Colin386/branch-ColinJavaDoc

Branch colin java doc

ninggggx99 committed:
Merge pull request #279 from ninggggx99/branch-DGupdate

Update for developer guide

matthewgani committed:
Edited UG and DG for extract and add

matthewgani committed:
Merge branch 'master' into branch-UpdateDevGuide

# Conflicts:
#	docs/DeveloperGuide.md

marcursor committed:
Add logging for check, done, undone, delete commands

ninggggx99 committed:
Fixing tiny errors

ninggggx99 committed:
Merge pull request #281 from ninggggx99/branch-DGupdate

Fixing tiny errors

marcursor committed:
Merge pull request #275 from matthewgani/branch-UpdateDevGuide

Added assertions and logging, fixes for add and extractCommand, updated UG and DG

marcursor committed:
Merge branch 'master' into branch-Logging

marcursor committed:
Merge pull request #283 from marcursor/branch-Logging

Branch logging

Colin386 committed:
Fix load save bug
Program will load all valid entries. Will report how many corrupted entries are present in the txt file

Colin386 committed:
Merge branch 'master' into branch-saveFix

Colin386 committed:
change checkstylemain

Colin386 committed:
pass storage load test

Colin386 committed:
change expected txt file

ninggggx99 committed:
- Fixed Deadline range
- Resolve Junit errors
- Make a common meesage for date and time exception

ninggggx99 committed:
Merge branch 'master' into branch-DeadlineDate

ninggggx99 committed:
Merge pull request #286 from ninggggx99/branch-DeadlineDate

Fixed Deadline range and junit for date time

ninggggx99 committed:
Merge pull request #285 from Colin386/branch-saveFix

Branch save fix

matthewgani committed:
Fixed logging and made addCommand cleaner

ninggggx99 committed:
Removed Logging files

matthewgani committed:
Merge branch 'master' into branch-AddCommandFix

# Conflicts:
#	src/main/java/seedu/duke/command/AddCommand.java

ninggggx99 committed:
Merge pull request #288 from ninggggx99/branch-RemoveLogging

Removed Logging files

Colin386 committed:
Repeat command format message
Repeat command format message will be printed out if the command arguments were given incorrectly

Colin386 committed:
Merge branch 'master' into branch-helpWrapAndException

matthewgani committed:
Fixed addCommand bug

matthewgani committed:
Merge branch 'master' into branch-AddCommandFix

Colin386 committed:
Add overflow exception
Events cannot be repeated more than 1000 times
Help command supports word wrapping
Help update for different formats of adding events

ninggggx99 committed:
Implemented error message with format

Colin386 committed:
checkstylemain

ninggggx99 committed:
Merge pull request #291 from ninggggx99/branch-ErrorMessage

Error message with format

Colin386 committed:
helpfile correction

marcursor committed:
Add format reminder to check, done, undone, delete command exception messages
Add done, undone, delete feature entries in Dev Guide
Update command format in command summary in User Guide

marcursor committed:
Add Java download link to UG

Reinbowl committed:
Update user guide

Reinbowl committed:
Merge pull request #293 from Reinbowl/userguide

Update user guide

Colin386 committed:
Merge pull request #292 from Colin386/branch-helpWrapAndException

Branch help wrap and exception

Colin386 committed:
Repeat command repairs
Fixes issues with repeat being asked to repeat for zero repetitions
Fixes printout of repeat command

Reinbowl committed:
Update PPP

matthewgani committed:
Fixed some inconsistencies in extract

Reinbowl committed:
Merge pull request #295 from Reinbowl/ppp

Update PPP

Colin386 committed:
Solve saving inaccuracies

matthewgani committed:
Bug fixes for add and extract and DG

Reinbowl committed:
Add page break

Reinbowl committed:
Merge pull request #296 from Reinbowl/ppp

Add page break to PPP

Reinbowl committed:
Add page breaks in DG

Reinbowl committed:
Merge pull request #297 from Reinbowl/developerguide

Add page breaks in DG

Reinbowl committed:
Update user guide

Reinbowl committed:
Add page breaks in UG

Reinbowl committed:
Update page break

Reinbowl committed:
Merge pull request #299 from Reinbowl/developerguide

Update page break in DG

Reinbowl committed:
Merge pull request #298 from Reinbowl/userguide

Update UG

Colin386 committed:
Merge branch 'master' into branch-setRepeatStatus

Reinbowl committed:
Update PPP

marcursor committed:
Add dev guide check, done, undone, delete argument tables
Edit PPP
Make CheckCommand getTime upper case the string before trying to parse it
Edit DateErrorException message date format in CheckCommand
Edit DateErrorException message date format in DateTimeParser
Edit done, undone, delete commands to handle excess semicolons

Reinbowl committed:
Merge pull request #301 from Reinbowl/ppp

Update PPP

marcursor committed:
Merge branch 'master' into branch-DevGuide

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/UserGuide.md
#	src/main/java/seedu/duke/command/CheckCommand.java
#	src/main/java/seedu/duke/parser/DateTimeParser.java
#	src/test/java/seedu/duke/command/CheckCommandTest.java
Conflicts resolved

marcursor committed:
Merge pull request #300 from Colin386/branch-setRepeatStatus

Branch set repeat status

Reinbowl committed:
Update page break in UG

Reinbowl committed:
Merge pull request #303 from Reinbowl/userguide

Update page break in UG

matthewgani committed:
Finished DG and fixed extractcommand bug

matthewgani committed:
Merge branch 'master' into branch-AddCommandFix

matthewgani committed:
Fixed checkstyle issues

ninggggx99 committed:
update developer guide with
- correct images
- table for arguments

ninggggx99 committed:
Merge branch 'master' into branch-DGrefine

ninggggx99 committed:
Fixed spelling errors

ninggggx99 committed:
Fix Typos

matthewgani committed:
Merge pull request #307 from ninggggx99/branch-DGrefine

Fixing Diagrams and adding table for arguments

ninggggx99 committed:
Fixing the diagram

ninggggx99 committed:
Fixing file extension

ninggggx99 committed:
Merge pull request #305 from matthewgani/branch-AddCommandFix

Branch add command fix

Colin386 committed:
add additonal object diagram

ninggggx99 committed:
Merge branch 'master' into branch-DGrefine

matthewgani committed:
Merge pull request #309 from ninggggx99/branch-DGrefine

Fixing Diagrams

ninggggx99 committed:
Added new features

ninggggx99 committed:
Update with bye

matthewgani committed:
Merge pull request #311 from ninggggx99/branch-PPPrefine

Added new features

matthewgani committed:
Merge pull request #313 from ninggggx99/branch-DGBye

DG bye command

Colin386 committed:
Additional help files added

Colin386 committed:
Merge branch 'master' into branch-DeveloperGuideV1

ninggggx99 committed:
- Developer guide intro added
- User guide edited

ninggggx99 committed:
Merge pull request #315 from ninggggx99/branch-DGintro

Minor changes made to DG and UG

ninggggx99 committed:
PPP update jar version

marcursor committed:
Update DG
Fix done, undone, delete command bug when trying to execute on a repeat task

marcursor committed:
Merge branch 'master' into branch-DevGuide

# Conflicts:
#	docs/DeveloperGuide.md
Resolved conflicts

marcursor committed:
Merge pull request #302 from marcursor/branch-DevGuide

Branch dev guide

ninggggx99 committed:
Update Junit with testing for invalid date range

ninggggx99 committed:
Merge pull request #317 from ninggggx99/branch-PPPVersion

PPP update jar version

ninggggx99 committed:
Merge pull request #319 from ninggggx99/branch-DeadlineJunitUpdate

Update Junit with testing for invalid date range

matthewgani committed:
Fixed PPP and 2 Sequence diagrams

matthewgani committed:
Merge branch 'master' into branch-PPPFinal

Colin386 committed:
Add dg changes
Added changes in help files, user guide and PPP file

Colin386 committed:
test push

Colin386 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

Colin386 committed:
Merge branch 'master' into branch-DeveloperGuideV1

Colin386 committed:
grave accent symbol

Colin386 committed:
escape grave

Colin386 committed:
Merge pull request #320 from Colin386/branch-DeveloperGuideV1

Branch developer guide v1

matthewgani committed:
Merge pull request #321 from matthewgani/branch-PPPFinal

Branch ppp final

ninggggx99 committed:
Conflict in pulling

ninggggx99 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

# Conflicts:
#	docs/diagrams/getDate_seq_diagram.jpg

matthewgani committed:
fixug

matthewgani committed:
qna fix

matthewgani committed:
QNA fix to use grave

marcursor committed:
Add diagrams to DG

marcursor committed:
Merge branch 'master' into branch-DevGuide

matthewgani committed:
Merge pull request #322 from matthewgani/branch-fixug

Branch fixug

marcursor committed:
Merge branch 'master' into branch-DevGuide

marcursor committed:
Merge pull request #323 from marcursor/branch-DevGuide

Branch dev guide

matthewgani committed:
Escape character for scheduler--

matthewgani committed:
Fix scheduler--

matthewgani committed:
Merge pull request #324 from matthewgani/branch-fixug

Branch fixug

Colin386 committed:
Developer guide updates

Colin386 committed:
Merge pull request #325 from Colin386/branch-dgupdate

Developer guide updates

matthewgani committed:
Fix some UG code not seen properly

Reinbowl committed:
Update DG

marcursor committed:
Add notes and warnings

Reinbowl committed:
Update DG

Reinbowl committed:
Update DG

marcursor committed:
Merge pull request #327 from Reinbowl/developerguide

Update DG

marcursor committed:
Merge pull request #328 from marcursor/branch-UserGuide

Add notes and warnings

matthewgani committed:
Warning for snippets not being exactly the same

marcursor committed:
Merge pull request #326 from matthewgani/branch-fixug

Fix some UG code not seen properly

matthewgani committed:
Remove unnecessary line from UG

marcursor committed:
Correct typo

marcursor committed:
Merge branch 'master' into branch-UserGuide

matthewgani committed:
Merge pull request #329 from matthewgani/branch-fixug

Remove unnecessary line from UG

marcursor committed:
Merge pull request #330 from marcursor/branch-UserGuide

Branch user guide

Colin386 committed:
last dg update

Colin386 committed:
Merge pull request #331 from Colin386/branch-dgupdatelast

last dg update

marcursor committed:
Update PPP

marcursor committed:
Set theme jekyll-theme-cayman

marcursor committed:
Delete _config.yml

marcursor committed:
no message

marcursor committed:
Merge branch 'master' of https://github.com/marcursor/tp

marcursor committed:
Fix PPP links

marcursor committed:
Merge pull request #332 from marcursor/branch-PPPfix

Fix PPP links

matthewgani committed:
Updated UG for CS2101, for add and extract commands.

matthewgani committed:
Merge pull request #333 from matthewgani/branch-2101UG

Updated UG for CS2101, for add and extract commands.

Colin386 committed:
repeat user guide change

Colin386 committed:
Merge branch 'master' into branch-UserGuideV3

Colin386 committed:
Merge pull request #334 from Colin386/branch-UserGuideV3

Branch user guide v3

marcursor committed:
Update command summary table format and convert to an image

marcursor committed:
Merge commit 'ad5d4f8fd18ed95e0f3adc84fd46ed302c13e5d9'

marcursor committed:
no message

marcursor committed:
Merge pull request #335 from marcursor/branch-UG

Update command summary table format and convert to an image

marcursor committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

ninggggx99 committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

ninggggx99 committed:
Updating ug with scenarios

ninggggx99 committed:
Merge pull request #336 from ninggggx99/branch-updateug

Updated UG with scenarios

ninggggx99 committed:
updating help

ninggggx99 committed:
view

Reinbowl committed:
Update UG

ninggggx99 committed:
removing extra a

matthewgani committed:
Fixed some full stops and small formatting issues, as well as 1.4

matthewgani committed:
Merge pull request #337 from ninggggx99/branch-updateug

updating help

matthewgani committed:
Merge pull request #338 from Reinbowl/userguide

Update UG

matthewgani committed:
Merge pull request #339 from matthewgani/branch-CS2101UG

Fixed some full stops, small formatting issues and 1.4

marcursor committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

matthewgani committed:
Edited extract

marcursor committed:
Update UG to use example usage format
- check, done, undone, delete commands

marcursor committed:
Correct config inconsistency in branch

marcursor committed:
Merge pull request #340 from marcursor/branch-UGv2

Update UG

Colin386 committed:
some ug changes

matthewgani committed:
Merge branch 'master' into branch-CS2101UG

matthewgani committed:
Merge pull request #341 from matthewgani/branch-CS2101UG

Edit UG

Colin386 committed:
more user guide changes

Colin386 committed:
Merge branch 'master' into branch-UserGuideV4

Colin386 committed:
Merge pull request #342 from Colin386/branch-UserGuideV4

Branch user guide v4

ninggggx99 committed:
deleted extra stuff

ninggggx99 committed:
Merge pull request #343 from ninggggx99/branch-ugfinal

Edited ug

Reinbowl committed:
Update UG

Reinbowl committed:
Merge pull request #344 from Reinbowl/userguide

Update UG

marcursor committed:
Update user guide

marcursor committed:
Add hyperlink

marcursor committed:
Add example usage intro sentence

marcursor committed:
Merge pull request #345 from marcursor/branch-UGv2

Updating UG

marcursor committed:
Fix command summary table border

marcursor committed:
Merge branch 'master' into branch-UGv2

matthewgani committed:
Fixed overall consistency in UG

matthewgani committed:
Merge branch 'master' into branch-CS2101UG

# Conflicts:
#	docs/UserGuide.md

marcursor committed:
Merge pull request #346 from marcursor/branch-UGv2

Fix command summary table border

matthewgani committed:
Merge branch 'master' into branch-CS2101UG

matthewgani committed:
Merge pull request #347 from matthewgani/branch-CS2101UG

Branch cs2101 UG

matthewgani committed:
Fixed some typos

matthewgani committed:
FIxed extract command scenario

ninggggx99 committed:
add scenario for reminder

Colin386 committed:
changes to save and bye

marcursor committed:
Update scenario for done

Colin386 committed:
Merge pull request #348 from ninggggx99/branch-ugfinal

add scenario for reminder

matthewgani committed:
Fixed formatting

Colin386 committed:
Merge branch 'master' into branch-UserGuideV5

matthewgani committed:
Merge branch 'master' into branch-CS2101UG

marcursor committed:
Merge branch 'master' of https://github.com/AY2021S1-CS2113T-T12-4/tp

matthewgani committed:
Merge pull request #349 from Colin386/branch-UserGuideV5

Branch user guide v5

matthewgani committed:
Merge pull request #350 from matthewgani/branch-CS2101UG

Branch cs2101 ug

marcursor committed:
Merge pull request #351 from marcursor/branch-UGv2

Update scenario

Colin386 committed:
bullet points added

Colin386 committed:
Merge pull request #352 from Colin386/branch-UserGuideV6

bullet points added
